agora.csl

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="sv-SE">
```

Comment:
@POBrien333, did you set this to Swedish intentionally?

It looks like the journal publishes in multiple Scandinavian languages, but the requestor mentioned Norwegian/norsk bokmål and the journal's domain is Norwegian too.